lib/hdk_records/src/metadata_helpers.rs

```diff
@@ -0,0 +1,186 @@
+use hdk::prelude::*;
+use crate::{ RecordAPIResult, DataIntegrityError };
+
+/// Metadata for a specific revision of a record, serializable for external transmission
+///
+#[derive(Clone, Serialize, Deserialize, SerializedBytes, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct RevisionMeta {
+    pub id: HeaderHash,
+    pub time: Timestamp,
+    pub author_id: AgentPubKey,
```

Comment:
is author_id as a property name part of the Valueflows spec?

Reply:
It is, but I take your point- there's no reason the two have to marry and actually this one doesn't anyway (it will have to be translated through a `whois` query against the REA Agent DNA in the GraphQL resolvers in order to be turned into an `AgentId` once #219 has been implemented). What would make more sense as idiomatically Holochain? `agent_pubkey`?

Reply:
Yeah something like... agent_pub_key to match the casing 👍